### PR TITLE
DF 1571 status not published

### PR DESCRIPTION
**Pull request by emina-str**
Added status "NOT_PUBLISHED" when status PUBLIC_POSTING and publicDisplay = false
manage-job-ad-detail

Showing on Table of Managed JobAd. We set the correct status in the table row, but in the filter we do not add this status, as that would require also backend changes.
manage-job-ad-table

### CODE REVIEW

File: alv-portal-ui/src/app/shared/backend-services/job-advertisement/job-advertisement.utils.ts [hunk 1]
```diff
@@ -22,4 +18,11 @@ export class JobAdvertisementUtils {
       && statusEnum !== JobAdvertisementStatus.CANCELLED;
   }
 
+  static resolveJobAdvertisementStatus(jobAdvertisement: JobAdvertisement): JobAdvertisementStatus {
+    if (jobAdvertisement.status === JobAdvertisementStatus.PUBLISHED_PUBLIC && !jobAdvertisement.publication.publicDisplay) {
```
Comment (seco-mgabor) [inline alv-portal-ui/src/app/shared/backend-services/job-advertisement/job-advertisement.utils.ts]: I'd put a comment here, to explain why the returned status is UNPUBLISHED, even if the job ad status is "PUBLISHED_PUBLIC".

Comment (cjpsycos) [reply alv-portal-ui/src/app/shared/backend-services/job-advertisement/job-advertisement.utils.ts]: it's just a temporary fix. next year come the real fix with the backend. so it's good for me at the moment

File: alv-portal-ui/src/app/shared/backend-services/job-advertisement/job-advertisement.utils.ts [hunk 2]
```diff
@@ -22,4 +18,11 @@ export class JobAdvertisementUtils {
...
+      return JobAdvertisementStatus.NOT_PUBLISHED;
+    }
```
Comment (tonim42) [inline alv-portal-ui/src/app/shared/backend-services/job-advertisement/job-advertisement.utils.ts]: The case of restricted ads is missing.

I would write the condition as follows:

 if (jobAdvertisement.status === JobAdvertisementStatus.PUBLISHED_PUBLIC && !jobAdvertisement.publication.publicDisplay) {
return jobAdvertisement.publication.restrictedDisplay ? JobAdvertisementStatus.PUBLISHED_RESTRICTED : JobAdvertisementStatus.NOT_PUBLISHED;
    }

Comment (emina-str) [reply alv-portal-ui/src/app/shared/backend-services/job-advertisement/job-advertisement.utils.ts]: Ok, will change it.

Comment (cjpsycos) [reply alv-portal-ui/src/app/shared/backend-services/job-advertisement/job-advertisement.utils.ts]: @tonim42 this was not agreed with sabrina. second this has nothing to do with restricted display. when a jobad is during the reporting-obligation period, it is already in status PUBLISHED_RESTRICTED.

Comment (emina-str) [reply alv-portal-ui/src/app/shared/backend-services/job-advertisement/job-advertisement.utils.ts]: @tonim42 , @cjpsycos  Sabrina said that change will not be implemented in this ticket.

**Pull request merged**